Assets/Treasure/TDK/Runtime/Common/TDK.Common.cs

```diff
@@ -36,7 +36,7 @@ public async Task<Transaction> WaitForTransaction(string queueId)
                     sw.Start();
                     while (sw.ElapsedMilliseconds < 2500)
                     {
-                        await Task.Yield();
+                        await Task.Yield(); // TODO any reason not to use `await Task.Delay(2500)`?
```

Comment:
If that does the same, let's do it - I probably copied this code block from somewhere